go-controller/hack/lint.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 GO111MODULE=on ${GOPATH}/bin/golangci-lint run \
+    --skip-dirs=pkg/crd/egressfirewall/v1/apis/ \
```

Comment:
Curious as to why lint check would be skipped for the `pkg/crd/egressfirewall/v1/apis/` directory

Reply:
Because those files are automatically generated by `controller-gen`/`client-gen`/`informer-gen` and the likes. They also happen to contain lines which do not adhere to our formatting, meaning every time anyone does `make codegen` they will have a broken patch that wasn't caused by them. Essentially: you can see (almost) everything in `pkg/crd` as anything in `vendor`, we use it but someone else wrote it.

Reply:
@alexanderConstantinescu Thanks for the explanation. 
@JacobTanenbaum it might be better to make an entry in the .golangci.yml below line at https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/.golangci.yml#L2 such as
```
run:
  skip-dirs:
    - vendor
    - pkg/crd/egressfirewall/v1/apis

  skip-files:
    - ".*\\_test.go$"
```

Reply:
Thanks! I didn't know that was the proper way to skip directories in the linter. Fixed